packages/cli/src/commands/createPlugin.ts

```diff
@@ -332,8 +332,16 @@ const createPlugin = async () => {
       type: 'input',
       name: 'id',
       message: chalk.blue('Enter an ID for the plugin [required]'),
-      validate: (value: any) =>
-        value ? true : chalk.red('Please enter an ID for the plugin'),
+      validate: (value: any) => {
+        if (!value) {
+          return chalk.red('Please enter an ID for the plugin');
+        } else if (!/^[-_a-zA-Z0-9]+$/.test(value)) {
```

Comment:
I think we could go with `/^[a-z0-9]+(-[a-z0-9]+)$/i`? i.e. enforcing kebab-case and no leading, trailing or repeated `-`

Reply:
but then the second parenthesis needs a `*` on it. sure i can do that